submissions/akulahere/index.html

```diff
@@ -0,0 +1,245 @@
+<!doctype html>
+<head>
+    <meta charset="utf-8">
+    <title>Interstellar</title>
+    <meta name="description" content="Interstellar booking">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/main.css">
+</head>
+<body>
+<header>
+    <a href="#" class="step-button">&#8249;</a>
+    <h1 class="header-name">
+        Select your places
+    </h1>
+    <a href="#" class="step-button">&#8250;</a>
+</header>
+<main class="main-container">
+    <section class="info-container">
```

Comment:
Fix aspect-ratio of poster on mobile and desktop.

![image](https://user-images.githubusercontent.com/16196199/107272645-ac935300-6a56-11eb-9cbc-13794f613246.png)